content/python/concepts/built-in-functions/id/id.md

```diff
@@ -0,0 +1,106 @@
+---
+Title: 'id()'
+Description: 'The id() function gives a unique number for any object in Python. This number is the location of the object in the computer’s memory.
```

Comment:
```suggestion
Description: 'Gives a unique number for any object in Python.'
```
Descriptions should be short, and function terms should start with a verb like "Returns" etc.